docker-compose.yml

```diff
@@ -27,6 +27,9 @@ services:
       - es
     volumes:
       - "./:/src/takwimu"
+    entrypoint:
+      - python
+    command: ["manage.py", "runserver", "0.0.0.0:8000"]
```

Comment:
These lines killed my morning! Was pulling my hair out "why does everything looks fine in the `Dockerfile` but the takwimu doesn't start?!!"

Sneaky @DavidLemayian. Very sneaky.

Reply:
😂  @kilemensi yeah, this is the better way to do development... particularly allows to edit `takwimu.scss` without having to run collectstatic.

What error are you getting?

Reply:
The `/docker-entrypoint.sh` wasn't running automatically @DavidLemayian. I lost count how many times I rebuilt the images :cry:

> particularly allows to edit `takwimu.scss` without having to run collectstatic.

This sounds cool actually. I'll definitely give it a try it.